### PR TITLE
feature/ADSCC-162

### PR DESCRIPTION
**Pull request by mantasheva-dzh**

### CODE REVIEW

File: src/main/java/io/vertx/kafka/admin/impl/KafkaAdminClientImpl.java [hunk 1]
```diff
@@ -564,4 +565,46 @@ private void describeLogDirsInternal(List<Integer> brokerIds, Promise<Map<Intege
       promise.fail(e);
     }
   }
```
Comment (mantasheva-dzh) [inline src/main/java/io/vertx/kafka/admin/impl/KafkaAdminClientImpl.java:567]: Changes were made due to describeLogDirsResult.all() method being deprecated after changing kafka library version

File: pom.xml [hunk 2]
```diff
@@ -30,7 +30,9 @@
 
   <properties>
     <stack.version>4.2.0-SNAPSHOT</stack.version>
-    <kafka.version>2.6.0</kafka.version>
+    <kafka.version>2.7.0</kafka.version>
```
Comment (mantasheva-dzh) [inline pom.xml:33]: Kafka version was changed due to DescribeFeatureResult being unavailable in previous version

Comment (mantasheva-dzh) [inline pom.xml:33]: Kafka version was changed due to  DescribeFeatureResult class being unavailable in previous version

File: src/main/java/io/vertx/kafka/admin/impl/KafkaAdminClientImpl.java (hunk 1 above)
Comment (mantasheva-dzh) [inline src/main/java/io/vertx/kafka/admin/impl/KafkaAdminClientImpl.java:567]: Changes in describeLogDirsInternal method were made due to describeLogDirsResult.all() method being deprecated after changing kafka library version

File: pom.xml [hunk 3]
```diff
@@ -68,6 +70,23 @@
...
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>${mockito.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.12.0</version>
```
Comment (VitekArkhipov) [inline pom.xml:88]: please, move to properties

Comment (mantasheva-dzh) [reply pom.xml]: My apologies. This library was used earlier. Now not needed. Removed it in the following commit

**Review by VitekArkhipov — DISMISSED** [review]

**Pull request closed without merging**